src/react-queries/queries/accountManagement.ts

```diff
@@ -55,27 +67,23 @@ export const accountManagement = createQueryKeys('accountManagement', {
         accountId: AccountId
         accountMetadata: AccountMetadata
         hasActivity: boolean
-      }[] = []
+      }[][] = []
```

Comment:
Is `accounts` still an appropriate name for what this evaluates to, now that it's "grouped" (an array of arrays)?

Reply:
well spotted, though this is removed completely in the following PR so will leave as-is to reduce conflicts